src/ts3interface.rs

```diff
@@ -64,19 +64,20 @@ pub unsafe fn private_init<T: Plugin>() -> Result<(), ::InitError> {
 // ************************** Interface for TeamSpeak **************************
 
 #[allow(non_snake_case)]
-#[no_mangle]
+#[unsafe(no_mangle)]
 #[doc(hidden)]
 pub extern "C" fn ts3plugin_apiVersion() -> c_int { 26 }
 
 #[allow(non_snake_case)]
-#[no_mangle]
+#[unsafe(no_mangle)]
 #[doc(hidden)]
 pub unsafe extern "C" fn ts3plugin_setFunctionPointers(funs: Ts3Functions) {
-	::TS3_FUNCTIONS = Some(funs);
+	let local = &crate::TS3_FUNCTIONS;
+	local.lock().unwrap().replace(funs);
```

Comment:
nit-pick: I guess the one-line variant still works?
```suggestion
	crate::TS3_FUNCTIONS.lock().unwrap().replace(funs);
```